doc/ref_arch/kubernetes/chapters/appendix-a.rst

```diff
@@ -17,9 +17,11 @@ the operating system kernel level.
 
 .. image:: ../figures/Model2-cluster-isolation.png
    :alt: "Cluster Isolation"
+   :name: "Cluster Isolation"
 
 .. image:: ../figures/Model1-ns.png
```

Comment:
should model 1 precede model 2? Model 1 and 2 are already in the image?

Also there is no caption under the two figures

Reply:
Done in [fea5914](https://github.com/cntt-n/CNTT/pull/3065/commits/fea59141befa0e4eb69b928e1807f3f6acab1556) captions are added in [995da2c](https://github.com/cntt-n/CNTT/pull/3065/commits/995da2c58e0b3b6fa3117a1a3d4fe9e1c5f3681d).